### PR TITLE
Add tweet for Operate First Data Science Community Meetup 4

### PR DESCRIPTION
**Pull request by aakankshaduggal**

### CODE REVIEW

File: tweets/2022-01/opf-ds-meetup4.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Learn how to automate your data science workflow and run your notebooks in an automated fashion by creating an #AI pipeline using #Elyra, #Kubeflow, and #Tekton. Join us on Jan 25th, 11:00ET for Operate First Data Science Meetup. Mark your calendars now - https://t.co/BUJzlVesCY
```
Comment (oindrillac) [inline tweets/2022-01/opf-ds-meetup4.tweet:1]:
```suggestion
Learn how to automate your data science workflow and run your notebooks in an automated fashion by creating an #AI pipeline using #Elyra, #Kubeflow, and #Tekton. Join us on Jan 25th, 11:00ET for the Operate First Data Science Meetup. Mark your calendars now https://t.co/BUJzlVesCY
```

Comment (aakankshaduggal) [reply tweets/2022-01/opf-ds-meetup4.tweet:1]: Thanks for the suggestion but this is exceeding the word limit.

Comment (oindrillac) [reply tweets/2022-01/opf-ds-meetup4.tweet:1]: I see. How about this?

Comment (aakankshaduggal) [reply tweets/2022-01/opf-ds-meetup4.tweet:1]: Looks good, thanks :)

**Review by oindrillac — APPROVED** [review]
Thanks @aakankshaduggal
/lgtm
/approve

**Pull request merged**